hladny_matfyzak.py

```diff
@@ -0,0 +1,57 @@
+from brutal.core.plugin import cmd
+from datetime import datetime
+import hladnymatfyzak
+
+
+DATE_FORMAT = '%d.%m.%Y'
+
+
+def validate_date_and_args(args):
+    """Validates given date"""
+    if len(args) == 1:
+        try:
+            valid = datetime.strptime(args[0], DATE_FORMAT)
+        except ValueError:
+            return None
+        return datetime(day=valid.day, month=valid.month, year=valid.year)
+    return datetime.today()
+
+
+def get_meals(event, restaurant):
+    """Returns string of meals from given restaurant"""
+    date_obj = validate_date_and_args(event.args)
+
+    if date_obj is None:
+        return 'Invalid date'
+
+    if restaurant == 'horna':
```

Comment:
@Adman this is all nice and everything, but don't we have more restaurants than just these two?